components/Chips/examples/ChipsSizingExampleViewController.m

```diff
@@ -79,12 +83,27 @@ - (void)viewDidLoad {
 - (void)viewWillLayoutSubviews {
   [super viewWillLayoutSubviews];
 
+  CGFloat topEdge;
+  if (@available(iOS 11, *)) {
+    topEdge = self.view.safeAreaInsets.top;
+  } else {
+    topEdge = self.topLayoutGuide.length;
+  }
+  CGRect frame = self.chipView.frame;
+  frame.origin.y = topEdge + 16;
+  self.chipView.frame = frame;
+
+  CGFloat contentBottomEdge = CGRectGetMaxY(self.chipView.frame);
+
   CGSize sliderSize = [self.widthSlider sizeThatFits:self.view.bounds.size];
-  self.widthSlider.frame = CGRectMake(20, 140, self.view.bounds.size.width - 40, sliderSize.height);
-  self.heightSlider.frame = CGRectMake(20, 140 + sliderSize.height + 20,
-                                       self.view.bounds.size.width - 40, sliderSize.height);
+  self.widthSlider.frame =
```

Comment:
Tip: Consider using constraint-based layouts. This is getting gnarly.

Reply:
Will leave as-is for this PR.